packages/hash/api/src/graphql/typeDefs/ontology/property-type.typedef.ts

```diff
@@ -0,0 +1,48 @@
+import { gql } from "apollo-server-express";
+
+export const propertyTypeTypedef = gql`
+  scalar PropertyType
+
+  type IdentifiedPropertyType {
+    propertyTypeVersionedUri: String!
+    createdBy: ID!
+    schema: PropertyType!
+    # TODO: we might need something like
+    # "referencedDataTypes: [DataType!]"
```

Comment:
Given the current naming, I think this would be
```suggestion
    # "referencedDataTypes: [IdentifiedDataType!]"
```

Reply:
Addressed in f4d46d43482d9b1f5a4ebbf268c876177a62aff1